src/main/java/frc/robot/subsystems/Launcher.java

```diff
@@ -0,0 +1,144 @@
+// Copyright (c) FIRST and other WPILib contributors.
+// Open Source Software; you can modify and/or share it under the terms of
+// the WPILib BSD license file in the root directory of this project.
+
+package frc.robot.subsystems;
```

Comment:
Another thing to do here is to make a DefaultLauncher command, which like the DefaultDrive command, uses input to set all of these.

Reply:
Working on it, check new commit called "Implementing Feedforward..."

Reply:
Pretty much done, check another commit called "added/changed joystick inputs"